### ISSUE TITLE
Not being able to manage IPv6 by protonvpn-cli may leak the system's IPv6 address.

### ISSUE THREAD

**Issue by saitam1** [ProtonVPN/protonvpn-cli#193]
System: Ubuntu 19.04
protonvpn-cli 1.1.2

I have followed this guide: https://protonvpn.com/support/linux-vpn-tool/
And I am getting the following error, when trying to connect to protonvpn network:
```
Connecting...
[!] Error connecting to VPN.
[!] There are issues in managing IPv6 in the system. Please test the system for the root cause.
Not being able to manage IPv6 by protonvpn-cli may leak the system's IPv6 address.
```
After installing `openresolv` and restart the computer, I could connect 2-3 times, but now suddenly I get again the above mentioned error all the time.

Is protonVPN made for Windows? Should I switch to Windows? Because I see a lot of people have problems to connect with Linux + protonvpn.

**Comment by Oops404** [comment]
same problem on my linux os

**Comment by cbdejavu** [comment]
Yeah, I've been using protonvpn for almost 2 years now and linux is just not a priority for them.  Unfortunately while I love their security the reliability and lack of support for linux is the reason I'm switching vpns when my subscription comes up next.
